bigchaindb/common/schema/__init__.py

```diff
@@ -0,0 +1,39 @@
+""" Schema validation related functions and data """
+from collections import OrderedDict
+import os.path
+
+import jsonschema
+import yaml
+
+from bigchaindb.common.exceptions import SchemaValidationError
+
+
+def ordered_load(stream):
+    """ Custom YAML loader to preserve key order """
+    class OrderedLoader(yaml.SafeLoader):
+        pass
+
+    def construct_mapping(loader, node):
+        loader.flatten_mapping(node)
+        return OrderedDict(loader.construct_pairs(node))
+    OrderedLoader.add_constructor(
+        yaml.resolver.BaseResolver.DEFAULT_MAPPING_TAG,
+        construct_mapping)
+    return yaml.load(stream, OrderedLoader)
+
+
+TX_SCHEMA_PATH = os.path.join(os.path.dirname(__file__), 'transaction.yaml')
+TX_SCHEMA_DATA = open(TX_SCHEMA_PATH).read()
```

Comment:
Also, maybe we should use an initialization function for initializing the schemas before we start all the processes.

This lets us preload the schema into every process, as well as throw an error and exit BigchainDB if the schemas aren't found.